### PR TITLE
Final version of the docs

### PR DESCRIPTION
**Pull request by julienduchesne**
Add some examples

### CODE REVIEW

File: template/extra_sprig.go [hunk 1]
```diff
@@ -7,25 +7,25 @@ import (
...
+	sprigRegex          = "Sprig Regex"
+	sprigSemver         = "Sprig Version comparison"
+	sprigString         = "Sprig Strings"
+	sprigReflect        = "Sprig Reflection"
+	sprigDefault        = "Sprig Default"
+	sprigOS             = "Sprig OS"
+	sprigTypeConversion = "Sprig Type Conversion"
+	sprigStringList     = "Sprig String Slice"
+	sprigURL            = "Sprig URL functions"
+	sprigNetwork        = "Sprig Network functions"
```
Comment (jocgir) [inline template/extra_sprig.go]: Why do you remove the links?

Comment (julienduchesne) [reply template/extra_sprig.go]: Links in menus or headings don't work well. A lot of them were wrong too

Comment (jocgir) [reply template/extra_sprig.go]: Initially, I was using them in the online doc. They were all working then 😁. Not sure that removing them was the best solution.

Comment (julienduchesne) [reply template/extra_sprig.go]: hmm. I will split on a comma and put the first part as the title and the second part under it. That way, we'll have title and link in the doc

**Pull request merged**